Modules/Filtering/LabelMap/test/itkLabelImageToLabelMapFilterTest.cxx

```diff
@@ -43,7 +43,8 @@ itkLabelImageToLabelMapFilterTest(int argc, char * argv[])
 
   if (argc != 1)
```

Comment:
This entire `if` block is unnecessary, because this test doesn't take any parameters.

Reply:
:+1: done in d06f2c6.